benchmarks/bench_methods_run.py

```diff
@@ -43,13 +39,17 @@
 # base string for the save names.
 base_name = 'run_0'
 # n_jobs for the parallel running of single core methods
-n_jobs = 1
+n_jobs = 30
```

Comment:
I would not define this globally. Actually, it would be nice to have multiple run of dicodile with various value of `n_jobs`. would it be possible to add this in the benchmark?
For instance with `n_jobs=5, 10, 30`?

this could be added in `methods_multivariate` using `functools.partial(run_multichannel_dicodile_fullrank, n_jobs=x)` from [here](https://docs.python.org/3/library/functools.html#functools.partial).

Reply:
I've changed n_jobs=1. For running dicodile with different values of n_jobs, I propose to do it in the following PR where we will improve parameters.